ajax/libs/botui/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "botui",
+  "description": "A JS library to build the UI for your bot",
+  "homepage": "https://botui.org",
+  "filename": "botui.min.js",
+  "author": {
+    "name": "Moin Uddin",
+    "url": "https://moin.im"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moinism/botui.git"
+  },
+  "keywords": [
+    "bot",
+    "chat",
+    "conversational",
+    "interface",
+    "js",
+    "ui"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/botui/botui.git",
+    "fileMap": [
+      {
+        "basePath": "",
```

Comment:
Let's use `build` as the basePath
because as you can see from upstream https://github.com/botui/botui/tree/master/build, the files we can use in web front-end is under this folder.
Thank you.